pkg/lib/serverTime.js

```diff
@@ -125,34 +125,17 @@ export function ServerTime() {
                     remote_offset = offsetms;
                     emit_changed();
                 })
-                .fail(function(ex) {
-                    console.log("Couldn't calculate server time offset: " + cockpit.message(ex));
-                });
+                .catch(ex => console.log("Couldn't calculate server time offset: " + cockpit.message(ex)));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19120-20230722-055749-01048432-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)